### DISCUSSION TITLE
Search (move to panel and limit scope)

### DISCUSSION THREAD

**Discussion by ghost** [VSCodium/vscodium#1732] [Q&A] [answered]
I'm trying out VSCodium and am trying to get things inline with how I work. When I do a search is replaces the Explorer "panel" (primary side bar?) which is kinda annoying due to how I have things sized. I'd rather it open on the "Panel" (that's the only term I've seen it called) that houses Problems, Debug, Terminal, etc. Even better if I could split that panel and anchor all search stuff over there so say Problems/Debug/Term above Search.

**Comment by daiyam** [comment, marked as answer]
click on the icon and drag the panel to where you want

**Reply by ghost** [reply]
Icon where? - Typed that then *tried* just clicking/dragging the title and it worked! i.e. "Icon" => text title heh. There is no visual cue (or icon) those things can be dragged or moved (perhaps my theme?) so I was kinda wondering if it was possible. I sourced the issues, read docs, searched the options...Most other things I've found have dedicated "Show on X | top | bottom" type things so I thought either it couldn't be done or I was missing something. Thanks!
